themes/default/content/docs/guides/continuous-delivery/azure-devops.md

```diff
@@ -97,6 +113,19 @@ Ubuntu agent, and some on a Windows agent. `pulumi` can be installed on these ag
 For the YAML-driven DevOps pipeline, the repository must contain the `azure-pipelines.yml` in the root of the repo for Azure DevOps to use it automatically.
 The following are samples only. You may choose to structure your configuration any way you like.
 
+### Log Pulumi Output as PR Comments {#log-pulumi-output-as-pr-comments}
```

Comment:
Just a thought -- what do you think about replacing this with a mention of the feature (and maybe link it to the [overview doc](https://github.com/pulumi/pulumi-az-pipelines-task/blob/master/overview.md#logs-output-as-pr-comments)) under the existing `Pulumi Task Extension for Azure Pipelines` section?

Reply:
I thought about this, but I'm generally not a fan of having to hop to links for "required" information in the document (as opposed to supplemental information). I don't think these steps will change over time and so keeping the doc and the overview file in synch shouldn't be a problem.

Reply:
@MitchellGerdisch sounds good :) I was thinking the same thing but thought it might be ok here.